plugins/acpi-facp/fu-plugin-acpi-facp.c

```diff
@@ -41,6 +41,12 @@ fu_plugin_acpi_facp_add_security_attrs(FuPlugin *plugin, FuSecurityAttrs *attrs)
 		return;
 	}
 	if (!fu_acpi_facp_get_s2i(facp)) {
+		FwupdBiosAttr *bios_attr =
+		    fu_context_get_bios_attr(fu_plugin_get_context(plugin), "SleepState");
```

Comment:
I think we want to get this by AppStream ID, no? e.g. the desired value from `com.lenovo.SleepState` might be `idle` and the desired value for `com.dell.SleepState` might be `s2idle`.

Reply:
Yes. This is only for Lenovo. Will fix to add the ID.